core/src/mindustry/content/UnitTypes.java

```diff
@@ -3767,7 +3767,7 @@ public static void load(){
             speed = 1.1f;
             rotateSpeed = 3.2f;
             accel = 0.1f;
-            health = 8000f;
+            health = 5500f;
```

Comment:
-3500 HP from quell? Are you sure?
btw i agree with all other changes, but this...

Reply:
i think beryllium walls are too cheap for such a high health so i decreased it greatly.i think 7200 is also ok if we build quells with tungsten walls